content/vi/docs/reference/glossary/cri-o.md

```diff
@@ -0,0 +1,27 @@
+---
+title: CRI-O
+id: cri-o
+date: 2020-03-05
+full_link: https://cri-o.io/#what-is-cri-o
+short_description: >
+  A lightweight container runtime specifically for Kubernetes
+  Một lightweight container runtime đặc biệt cho Kubernetes
+aka:
+tags:
+- tool
+---
+Một công cụ giúp bạn sử dụng các OCI container runtime với Kubernetes CRI.
+
+<!--more-->
+
+CRI-O is an implementation of the {{< glossary_tooltip term_id="cri" >}}
+to enable using {{< glossary_tooltip text="container" term_id="container" >}}
+runtimes that are compatible with the Open Container Initiative (OCI)
+[runtime spec](http://www.github.com/opencontainers/runtime-spec).
+
+Deploying CRI-O allows Kubernetes to use any OCI-compliant runtime as the container
+runtime for running {{< glossary_tooltip text="Pods" term_id="pod" >}}, and to fetch
+OCI container images from remote registries.

```

Comment:
The English is still here.

Reply:
Thank you. I am sorry for this mistake. I will update

Reply:
@sftim these words are primitive and no need to translate to Vietnamese. Thank you for your review.